helloworld/main.py

```diff
@@ -2,14 +2,18 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 from colors import green
+
 from helloworld.greet.greeting import Greeter
+from helloworld.protos.config_pb2 import Config
+from helloworld.util.config_loader import load_config_from_json
 
 
-def say_hello():
-    greeter = Greeter()
-    sentence = greeter.greet("world")
+def say_hello(config: Config) -> None:
+    greeter = Greeter(greetings=config.greetings, languages=config.languages)
+    sentence = greeter.greet("world").capitalize()
     print(green(sentence))
 
 
 if __name__ == "__main__":
-    say_hello()
+    config = load_config_from_json("helloworld", "config.json")
```

Comment:
Normally, `helloworld` would be replaced with `__name__`. But we know that `__name__` will have been overridden to `__main__`, so that doesn't work properly.

Lmk please if you know of a way to programmatically find that the module name is `helloworld`.